docs/self-hosting/setup/one-click.mdx

```diff
@@ -276,7 +276,7 @@ Y
  ✔ Container formbricks-postgres-1    Started                                                                                                                                                    1.0s
  ✔ Container formbricks-formbricks-1  Started                                                                                                                                                    1.6s
  ✔ Container traefik                  Started                                                                                                                                                    2.8s
-🔗 To edit more variables and deeper config, go to the formbricks/docker-compose.yml, edit the file, and restart the container!
+🔗 To edit more variables and deeper config, go to the formbricks/docker/docker-compose.yml, edit the file, and restart the container!
```

Comment:
The previous information was correct. This doesn't follow our repository structure, but the one click script creates a `formbricks` directory on the server and downloads the `docker-compose.yml` into this.